cmake/patches/openvino/ngraph_protobuf.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/cmake/external_protobuf.cmake b/cmake/external_protobuf.cmake
+index 9d5111b78..bd8a1c12e 100644
+--- a/cmake/external_protobuf.cmake
++++ b/cmake/external_protobuf.cmake
+@@ -44,7 +44,7 @@ if(CMAKE_CROSSCOMPILING)
+         message(FATAL_ERROR "System Protobuf is needed while cross-compiling")
+     endif()
+ else()
+-    set(PROTOC_VERSION "3.7.1")
++    set(PROTOC_VERSION "3.11.3")
```

Comment:
can we add a comment here that the version here needs to be consistent with the version of protobuf that OnnxRuntime uses?
this is very important.